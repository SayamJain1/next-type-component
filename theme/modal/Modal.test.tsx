import React from "react";
import renderer from "react-test-renderer";
import Modal from "./Index";

it("performs snapshot testing", () => {
  const tree = renderer.create(<Modal />).toJSON();
  expect(tree).toMatchSnapshot();
});
