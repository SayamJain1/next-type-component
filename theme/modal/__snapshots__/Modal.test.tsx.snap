// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`performs snapshot testing 1`] = `null`;
